lib/jsonapi/rails/renderer.rb

```diff
@@ -33,6 +33,7 @@ def rails_renderer(renderer)
         reverse_mapping = request.env[ActionController::REVERSE_MAPPING_KEY]
         options = options.merge(_reverse_mapping: reverse_mapping)
         json = renderer.render(json, options) unless json.is_a?(String)
+        json = json.to_json unless json.is_a?(String)
```

Comment:
You guys are obviously right. How about pinning `jsonapi-serializable` to `>= 0.1.3`?